gradle/spotbugs.gradle

```diff
@@ -10,27 +10,24 @@ allprojects {
     toolVersion = '4.1.3'
   }
 
-  if (!isCI) {
-    spotbugsTest {
-      reports {
-        html.enabled = true
-        xml.enabled = false
-        html {
-          destination = file("$buildDir/reports/spotbugs/main.html")
-          stylesheet = 'fancy-hist.xsl'
-        }
+  spotbugsTest {
+    reports {
+      html.enabled = !isCI
+      xml.enabled = isCI
+      html {
+        destination = file("$buildDir/reports/spotbugs/main.html")
+        stylesheet = 'fancy-hist.xsl'
       }
     }
-    spotbugsMain {
-      reports {
-        html.enabled = true
-        xml.enabled = false
-        html {
-          destination = file("$buildDir/reports/spotbugs/main.html")
-          stylesheet = 'fancy-hist.xsl'
-        }
+  }
+  spotbugsMain {
+    reports {
+      html.enabled = !isCI
+      xml.enabled = isCI
+      html {
```

Comment:
If these are default values anyway, then remove them

Reply:
Default is `true` for `xml.enabled`. However, to enable HTML reports, you have to explicitly disable the XML ones so both need to be present and set. Does this make sense or have I missed your point entirely?

Reply:
I was talking about `html` configuration for stylesheet and report location. Why default values for them are not enough?

Reply:
Oh I follow. Yeah we can remove the report location, but maybe keep the stylesheet, as it's not the default?